build/update-olm.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import yaml
+
+
+_ANNOTATIONS = {
+    'categories': 'Virtualization',
+    'description': \
+        'Node-maintenance-operator maintains nodes in cluster',
+}
+_DESCRIPTION = "Node maintenance operator"
+_NAMESPACE = 'default'
+_SPEC = {
+    'description': _DESCRIPTION,
+    'provider': {
+        'name': 'KubeVirt project'
+    },
+    'maintainers': [{
+        'name': 'KubeVirt project',
+        'email': 'kubevirt-dev@googlegroups.com',
+    }],
+    'keywords': [
+        'KubeVirt', 'Virtualization', 'Node-maintenance'
+    ],
+    'links': [{
+        'name': 'KubeVirt',
+        'url': 'https://kubevirt.io',
+    }, {
+        'name': 'Source Code',
+        'url': 'https://github.com/kubevirt/node-maintenance-operator'
+    }],
+    'labels': {
+        'alm-owner-kubevirt': 'nodemaintenanceoperator',
+        'operated-by': 'nodemaintenanceoperator',
+    },
+    'selector': {
+        'matchLabels': {
+            'alm-owner-kubevirt': 'nodemaintenanceoperator',
+            'operated-by': 'nodemaintenanceoperator',
+        },
+    },
+}
+
+_CRD_INFOS = {
+    'nodemaintenances.kubevirt.io': {
+        'displayName': 'KubeVirt node maintenance',
+        'description': \
+                'Represents a deployment of node maintenance',
+        'specDescriptors': [{
+            'description': \
+                'The version of the node maintenance to deploy',
+            'displayName': 'Version',
+            'path': 'version',
+            'x-descriptors': [
+                'urn:alm:descriptor:io.kubernetes.node-maintenance:version',
+            ],
+        }],
+    }
+}
```

Comment:
mostly I was following what others did and added something sensible(-ish), I guess that's good enough here too.